src/GuildsApp.tsx

```diff
@@ -38,58 +39,60 @@ const GuildsApp = () => {
       <HashRouter basename="/guilds">
         <GlobalErrorBoundary>
           <WalletWeb3Manager>
-            <GlobalStyle />
-            <Header />
-            <Container>
-              <Switch>
-                <Redirect
-                  exact
-                  from="/"
-                  to="/rinkeby/0x9cdc16b5f95229b856cba5f38095fd8e00f8edef"
-                />
-                <Redirect
-                  exact
-                  from="/:chain_name"
-                  to="/:chain_name/0x9cdc16b5f95229b856cba5f38095fd8e00f8edef"
-                />
-                <Route exact path="/:chain_name/:guild_id">
-                  <GuildsContextProvider>
+            <TransactionModalProvider>

```

Comment:
Is there a reason this is above Global style?

Reply:
no, I just picked a spot and slotted it in. I'll place it somewhere else